opentaskspal/src/main/java/org/dmfs/opentaskspal/rowsets/Subtasks.java

```diff
@@ -16,29 +16,28 @@
 
 package org.dmfs.opentaskspal.rowsets;
 
-import android.content.ContentProviderClient;
-import android.support.annotation.NonNull;
-
+import org.dmfs.android.contentpal.RowReference;
 import org.dmfs.android.contentpal.RowSet;
-import org.dmfs.android.contentpal.RowSnapshot;
+import org.dmfs.android.contentpal.View;
 import org.dmfs.android.contentpal.predicates.ReferringTo;
 import org.dmfs.android.contentpal.rowsets.DelegatingRowSet;
 import org.dmfs.android.contentpal.rowsets.QueryRowSet;
-import org.dmfs.opentaskspal.views.TasksView;
-import org.dmfs.tasks.contract.TaskContract;
+import org.dmfs.tasks.contract.TaskContract.Tasks;
 
 
 /**
  * {@link RowSet} for the subtasks of a given task.
  *
  * @author Gabor Keszthelyi
  */
-public final class Subtasks extends DelegatingRowSet<TaskContract.Tasks>
+public final class Subtasks extends DelegatingRowSet<Tasks>
 {
+    public static final String[] PROJECTION = { Tasks.PARENT_ID };
```

Comment:
What's this `PROJECTION` good for? Isn't it rather pointless? When you're using this you already know that all rows will have the same `PARENT_ID` value, i.e. the value of the parent task `_ID`.
The selection columns don't have to be part of the projection in order to work.

Reply:
Right, it was wrong here, `PARENT_ID` is not part of the projection here. Removed it.